openapi.yml

```
@@ -114,24 +130,27 @@ paths:
             type: string
           description: Maintenance window ID
       responses:
-        '204':
+        '200':
           description: Maintenance window deleted.
         '400':
           description: Invalid JSON.
         '404':
           description: Maintenance window not found.
-  '/maintenance_window/{mw_id}/end':
+  '/maintenance/{mw_id}/end':
     patch:
       tags:
         - Update
       summary: Finish a maintenance window right now
       responses:
```

The parameter "mw_id" needs to be defined as a path parameter;
```suggestion
      summary: Finish a maintenance window right now
      parameters:
        - name: mw_id
          in: path
          required: true
          schema:
            type: string
          description: Maintenance window ID
      responses:
```